packages/accordion/src/Accordion.stories.tsx

```diff
@@ -16,25 +16,19 @@ export default {
 			options: ['body', 'bodyAlt'],
 			control: { type: 'select' },
 		},
-		boxPalette: {
-			options: ['light', 'dark'],
-			control: { type: 'select' },
-		},
 	},
 } as ComponentMeta<typeof Accordion>;
 
 type AccordionExampleProps = {
 	boxBackground: 'body' | 'bodyAlt';
-	boxPalette: 'light' | 'dark';
 };
 
 const AccordionBasicExample: Story<AccordionExampleProps> = ({
 	boxBackground,
```

Comment:
Hey @nathsimpson, this `boxBackground` property is being used for two things

1) The background of the story
2) The background colour of the accordion item

Can we just rename this to `background`  and *only* use it for the `AccordionItem`? The background of the story should be configurable with the new dropdown in your other PR.

Reply:
background of story will not be configurable in Storybook